geometry_msgs/Point top_left
geometry_msgs/Point width_height